docs/getting_started.md

```diff
@@ -106,7 +106,7 @@ EOF
 
 Once the BootstrapKubeconfig CR is created, fetch the object to copy the bootstrap kubeconfig file details from the Status field
 ```shell
-kubectl get bootstrapkubeconfig bootstrap-kubeconfig -n default -o json | jq '.status'
+kubectl get bootstrapkubeconfig bootstrap-kubeconfig -n default -o=jsonpath='{.status.bootstrapKubeconfigData}' > ~/bootstrap-kubeconfig
```

Comment:
What is the reason for storing the  `bootstrapKubeconfigData` first in `~/bootstrap-kubeconfig` and then copying the content to `~/bootstrap-kubeconfig.conf`?

Reply:
Right. Updated.